osmosis-1/osmosis.zone_assets.json

```diff
@@ -940,6 +940,20 @@
       "path": "transfer/channel-258/mO9W",
       "osmosis_verified": false
     },
+    {
+      "chain_name": "odin",
+      "base_denom": "udoki",
+      "path": "transfer/channel-258/udoki",
+      "osmosis_verified": false,
+      "osmosis_unlisted": true
+    },
+    {
```

Comment:
Please move this addition to the bottom of the list

Reply:
ok! Will do

Reply:
This one is fixed